### PR TITLE
Fixes #15515 - Fix undefined method get_humanized for nil:NilClass

### PR DESCRIPTION
**Pull request by vision2910**

### CODE REVIEW

**Comment by vision2910** [comment]
The above fix will prevent the exception being raised while invoking format_task_input method with nil param in the below files-

[/views/foreman_tasks/recurring_logics/index.html.erb](https://github.com/theforeman/foreman-tasks/blob/master/app/views/foreman_tasks/recurring_logics/index.html.erb#L21)

[/views/foreman_tasks/task_groups/recurring_logic_task_groups/_recurring_logic_task_group.html.erb](https://github.com/theforeman/foreman-tasks/blob/master/app/views/foreman_tasks/task_groups/recurring_logic_task_groups/_recurring_logic_task_group.html.erb#L13)

**Comment by iNecas** [comment]
[test]

File: test/helpers/foreman_tasks/tasks_helper_test.rb [hunk 1]
```diff
@@ -15,6 +15,9 @@ class TasksHelperTest < ActionView::TestCase
         format_task_input(@task, true).must_equal("Create user 'Anonymous Admin'")
       end
 
+      it 'displays the dash if task is nul' do
```
Comment (iNecas) [inline test/helpers/foreman_tasks/tasks_helper_test.rb]: nitpick `if the task is nil`

**Comment by iNecas** [comment]
@theforeman-bot complains about the commit message size. Not that I like this policy too much, but still, let's make him happy :)

**Comment by iNecas** [comment]
[test]

**Comment by dLobatog** [comment]
@iNecas keep in mind you can cherry-pick & `commit --amend` to fix it in these situations :smile:

**Comment by iNecas** [comment]
I know but I preffer to teach how to fish instead :)

**Comment by iNecas** [comment]
I will not block this on just on the lenght of the commit message (I've updated that while merging). Thanks @vision2910

**Pull request merged**